src/renderers/webgl/WebGLState.js

```diff
@@ -339,6 +346,47 @@ THREE.WebGLState = function ( gl, paramThreeToGL ) {
 
 	};
 
+	this.activeTexture = function ( webglSlot ) {
+
+		webglSlot = webglSlot || ( gl.TEXTURE0 + maxTextures - 1 );
```

Comment:
If `webglSlot` is `0` then we would be getting `gl.TEXTURE0 + maxTextures - 1`. No?

Maybe this is what you're after?

```javascript
if ( webglSlot === undefined ) webglSlot = gl.TEXTURE0 + maxTextures - 1;
```